src/01-background/README.md

```diff
@@ -3,43 +3,53 @@
+that you live and work in at a comfortable temperature, and control the components that make the
+vehicles you travel in stop and go.
 
-The main trait of these systems is that they operate without user intervention even if they expose a
-user interface like a washing machine does; most of their operation is done on their own.
+Most embedded systems operate without user intervention. Even if they expose a user interface like a
+washing machine does; most of their operation is done on their own.
 
-The other common trait of these systems is that they *control* a process. And for that these systems
-usually have one or more sensors and one or more actuators. For example, an HVAC system has several
-sensors, thermometers and humidity sensors spread across some area, and several actuators as well,
-heating elements and fans connected to ducts.
+Embedded systems are often used to *control* a physical process. To make this possible, they have
+one or more devices to tell them about the state of the world ("sensors"), and one or more
+devices which allow them to change things ("actuators"). For example, a building climate control
+system might have:
+
+- Sensors which measure temperature and humidity in various locations.
+- Actuators which control the speed of fans.
+- Actuators which cause heat to be added or removed from the building.
 
 ## When should I use a microcontroller?
 
-All these application I've mentioned, you can probably implement with a Raspberry Pi, a computer
-that runs Linux. Why should I bother with a microcontroller that operates without an OS? Sounds like
-it would be harder to develop a program.
+Many of the embedded systems listed above could be implemented with a computer running Linux (for
+example a "Raspberry Pi"). Why use a microcontroller instead? Sounds like it might be harder to
+develop a program.
+
+Some reasons might include:
+
+**Cost.** A microcontroller is much cheaper than a general purpose computer. Not only is the
+microcontroller cheaper; it also requires many fewer external electrical components to operate.
+This makes Printed Circuit Boards (PCB) smaller and cheaper to design and manufacture.
 
-The main reason is cost. A microcontroller is much cheaper than a general purpose computer. Not only
-the microcontroller is cheaper; it also requires many fewer external electrical components to
-operate. This makes Printed Circuit Boards (PCB) smaller and cheaper to design and manufacture.
+**Power consumption.** Most microcontrollers consume a fraction of the power of a full blown
+processor. For applications which run on batteries, that makes a huge difference.
 
-The other big reason is power consumption. A microcontroller consumes orders of magnitude less power
-than a full blown processor. If your application will run on batteries that makes a huge difference.
+**Responsiveness.** To accomplish their purpose reliably, some embedded systems must always react
```

Comment:
It would be cool if we could at least mention the term "real time" her so people have at least heard it, after all it's exactly the thing you are describing here.

Reply:
ACK